SimpleDoTTest tests using external servers is flaky

<!-- Hi! Thanks for filing an issue. It will be read with care by human beings. Can we ask you to please fill out this template and not simply demand new features or send in complaints? Thanks! -->
<!-- Also please search the existing issues (both open and closed) to see if your report might be duplicate -->
- [x] This is not a support question, I have read [about opensource](https://www.powerdns.com/opensource.html) and will send support questions to the IRC channel, [GitHub Discussions](https://github.com/PowerDNS/pdns/discussions/) or the mailing list.
- [x] I have read and understood the ['out in the open' support policy](https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/)

<!-- Tell us what is issue is about -->
 - Program: Authoritative, Recursor, dnsdist
 - Issue type: Bug report

### Short description
Given that github action runner networking is "known flaky", I'd suggest changing the timeout for CI tests that deal with external servers from 2s (I'm told this is the python test default) to 3s?

### Environment
<!-- Tell us about the environment -->
 - Operating system: ubuntu-22.04
 - Software version: master 50c0dcca1f0a652abac4044c966496a91af2e816
 - Software source: PowerDNS repository, compiled by GitHub runner

### Steps to reproduce
<!-- Tell us step-by-step how the issue can be triggered. Please include your configuration files and any (Lua) scripts that are loaded. -->
1. Let CI run for a merge
2. Check tests for failures
3. See failure in https://github.com/PowerDNS/pdns/actions/runs/11703588629/job/32594551185#step:10:1464

### Expected behaviour
<!-- What would you expect to happen when the reproduction steps are run -->

CI tests shouldn't be flaky

### Actual behaviour

```
2024-11-06T12:48:16.0718946Z =================================== FAILURES ===================================
2024-11-06T12:48:16.0720585Z ____________________________ SimpleDoTTest.testTXT _____________________________
2024-11-06T12:48:16.0721203Z 
2024-11-06T12:48:16.0721521Z self = <test_SimpleDoT.SimpleDoTTest testMethod=testTXT>
2024-11-06T12:48:16.0722044Z 
2024-11-06T12:48:16.0728111Z     def testTXT(self):
2024-11-06T12:48:16.0729760Z         expected = dns.rrset.from_text('dot-test-target.powerdns.org.', 0, dns.rdataclass.IN, 'TXT', 'https://github.com/PowerDNS/pdns/pull/12825')
2024-11-06T12:48:16.0731842Z         query = dns.message.make_query('dot-test-target.powerdns.org', 'TXT', want_dnssec=True)
2024-11-06T12:48:16.0733092Z         query.flags |= dns.flags.AD
2024-11-06T12:48:16.0733619Z     
2024-11-06T12:48:16.0734051Z         res = self.sendUDPQuery(query)
2024-11-06T12:48:16.0734580Z     
2024-11-06T12:48:16.0734939Z >       self.assertMessageIsAuthenticated(res)
2024-11-06T12:48:16.0735358Z 
2024-11-06T12:48:16.0735530Z test_SimpleDoT.py:40: 
2024-11-06T12:48:16.0737979Z _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2024-11-06T12:48:16.0738561Z 
2024-11-06T12:48:16.0738944Z self = <test_SimpleDoT.SimpleDoTTest testMethod=testTXT>, msg = None
2024-11-06T12:48:16.0739544Z 
2024-11-06T12:48:16.0739801Z     def assertMessageIsAuthenticated(self, msg):
2024-11-06T12:48:16.0740496Z         """Asserts that the message has the AD bit set
2024-11-06T12:48:16.0741082Z     
2024-11-06T12:48:16.0741534Z         @param msg: the dns.message.Message to check"""
2024-11-06T12:48:16.0742145Z     
2024-11-06T12:48:16.0742546Z         if not isinstance(msg, dns.message.Message):
2024-11-06T12:48:16.0743001Z >           raise TypeError("msg is not a dns.message.Message")
2024-11-06T12:48:16.0743457Z E           TypeError: msg is not a dns.message.Message
2024-11-06T12:48:16.0743754Z 
2024-11-06T12:48:16.0743864Z recursortests.py:931: TypeError
2024-11-06T12:48:16.0744386Z ---------------------------- Captured stdout setup -----------------------------
2024-11-06T12:48:16.0744807Z Setting up UDP socket..
2024-11-06T12:48:16.0745070Z Launching pdns_recursor..
2024-11-06T12:48:16.0745845Z /opt/pdns-recursor/sbin/pdns_recursor --config-dir=configs/SimpleDoT --local-port=5300 --security-poll-suffix= --enable-old-settings
2024-11-06T12:48:16.0746624Z =============================== warnings summary ===============================
2024-11-06T12:48:16.0747197Z .venv/lib/python3.11/site-packages/pysnmp/carrier/asyncore/base.py:9
2024-11-06T12:48:16.0748989Z   /__w/pdns/pdns/regression-tests.recursor-dnssec/.venv/lib/python3.11/site-packages/pysnmp/carrier/asyncore/base.py:9: DeprecationWarning: The asyncore module is deprecated and will be removed in Python 3.12. The recommended replacement is asyncio
2024-11-06T12:48:16.0750445Z     import asyncore
2024-11-06T12:48:16.0750600Z 
2024-11-06T12:48:16.0750911Z -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
2024-11-06T12:48:16.0751612Z - generated xml file: /__w/pdns/pdns/regression-tests.recursor-dnssec/pytest.xml -
2024-11-06T12:48:16.0752205Z =========================== short test summary info ============================
2024-11-06T12:48:16.0752912Z FAILED test_SimpleDoT.py::SimpleDoTTest::testTXT - TypeError: msg is not a dns.message.Message
2024-11-06T12:48:16.0753604Z ======= 1 failed, 463 passed, 2 skipped, 1 warning in 501.53s (0:08:21) ========
2024-11-06T12:48:16.1658732Z + EXIT=1
2024-11-06T12:48:16.1659165Z + ./printlogs.py
2024-11-06T12:48:16.1900202Z ==============> configs/SimpleDoT/recursor.log <==============
```
...
```
2024-11-06T12:48:16.2019188Z Nov 06 12:47:34 [1] org: Checking if we have NS in cache for '.'
2024-11-06T12:48:16.2020702Z Nov 06 12:47:34 [1] org: NS (with ip, or non-glue) in cache for '.' -> 'a.root-servers.net', within bailiwick: 1, in cache, ttl=3600000
2024-11-06T12:48:16.2022640Z Nov 06 12:47:34 [1] org: NS (with ip, or non-glue) in cache for '.' -> 'b.root-servers.net', within bailiwick: 1, in cache, ttl=3600000
2024-11-06T12:48:16.2024650Z Nov 06 12:47:34 [1] org: NS (with ip, or non-glue) in cache for '.' -> 'c.root-servers.net', within bailiwick: 1, in cache, ttl=3600000
2024-11-06T12:48:16.2026601Z Nov 06 12:47:34 [1] org: NS (with ip, or non-glue) in cache for '.' -> 'd.root-servers.net', within bailiwick: 1, in cache, ttl=3600000
2024-11-06T12:48:16.2028532Z Nov 06 12:47:34 [1] org: NS (with ip, or non-glue) in cache for '.' -> 'e.root-servers.net', within bailiwick: 1, in cache, ttl=3600000
2024-11-06T12:48:16.2029886Z Nov 06 12:47:34 [1] org: NS (with ip, or non-glue) in cache for '.' -> 'f.root-servers.net', within bailiwick: 1, in cache, ttl=3600000
2024-11-06T12:48:16.2030943Z Nov 06 12:47:34 [1] org: NS (with ip, or non-glue) in cache for '.' -> 'g.root-servers.net', within bailiwick: 1, in cache, ttl=3600000
2024-11-06T12:48:16.2031984Z Nov 06 12:47:34 [1] org: NS (with ip, or non-glue) in cache for '.' -> 'h.root-servers.net', within bailiwick: 1, in cache, ttl=3600000
2024-11-06T12:48:16.2033057Z Nov 06 12:47:34 [1] org: NS (with ip, or non-glue) in cache for '.' -> 'i.root-servers.net', within bailiwick: 1, in cache, ttl=3600000
2024-11-06T12:48:16.2034107Z Nov 06 12:47:34 [1] org: NS (with ip, or non-glue) in cache for '.' -> 'j.root-servers.net', within bailiwick: 1, in cache, ttl=3600000
2024-11-06T12:48:16.2035171Z Nov 06 12:47:34 [1] org: NS (with ip, or non-glue) in cache for '.' -> 'k.root-servers.net', within bailiwick: 1, in cache, ttl=3600000
2024-11-06T12:48:16.2036224Z Nov 06 12:47:34 [1] org: NS (with ip, or non-glue) in cache for '.' -> 'l.root-servers.net', within bailiwick: 1, in cache, ttl=3600000
2024-11-06T12:48:16.2037253Z Nov 06 12:47:34 [1] org: NS (with ip, or non-glue) in cache for '.' -> 'm.root-servers.net', within bailiwick: 1, in cache, ttl=3600000
2024-11-06T12:48:16.2038264Z Nov 06 12:47:34 [1] org: We have NS in cache for '.' (flawedNSSet=0)
2024-11-06T12:48:16.2038817Z Nov 06 12:47:34 [1] org: Cache consultations done, have 13 NS to contact
2024-11-06T12:48:16.2039657Z Nov 06 12:47:34 [1] org: Nameservers: c.root-servers.net(0.00ms), h.root-servers.net(0.00ms), a.root-servers.net(0.00ms), 
2024-11-06T12:48:16.2040749Z Nov 06 12:47:34 [1]              g.root-servers.net(0.00ms), f.root-servers.net(0.00ms), k.root-servers.net(0.00ms), 
2024-11-06T12:48:16.2041636Z Nov 06 12:47:34 [1]              d.root-servers.net(0.00ms), m.root-servers.net(0.00ms), b.root-servers.net(0.00ms), 
2024-11-06T12:48:16.2042512Z Nov 06 12:47:34 [1]              j.root-servers.net(0.00ms), i.root-servers.net(0.00ms), l.root-servers.net(0.00ms), 
2024-11-06T12:48:16.2043258Z + exit 1
2024-11-06T12:48:16.2043596Z Nov 06 12:47:34 [1]              e.root-servers.net(0.00ms)
2024-11-06T12:48:16.2044155Z Nov 06 12:47:34 [1] org: Trying to resolve NS 'c.root-servers.net' (1/13)
2024-11-06T12:48:16.2044879Z Nov 06 12:47:34 [1] c.root-servers.net: Nameserver c.root-servers.net IPs: 192.33.4.12(0.00ms)
2024-11-06T12:48:16.2045580Z Nov 06 12:47:34 [1] org: Resolved '.' NS c.root-servers.net to: 192.33.4.12
2024-11-06T12:48:16.2046162Z Nov 06 12:47:34 [1] org: Trying IP 192.33.4.12:53, asking 'org|A'
2024-11-06T12:48:16.2046645Z Nov 06 12:47:36 [1] org: Timeout resolving after 1500.32 ms 
```
...
```
2024-11-06T12:48:16.2473009Z Nov 06 12:47:36 [1] pdns-public-ns1.powerdns.com: Nameserver pdns-public-ns1.powerdns.com IPs: 188.166.104.87(0.00ms)
2024-11-06T12:48:16.2474075Z Nov 06 12:47:36 [1] dot-test-target.powerdns.org: Resolved 'powerdns.org' NS pdns-public-ns1.powerdns.com to: 188.166.104.87
2024-11-06T12:48:16.2475148Z Nov 06 12:47:36 [1] dot-test-target.powerdns.org: Trying IP 188.166.104.87:53, asking 'dot-test-target.powerdns.org|TXT'
2024-11-06T12:48:16.2476023Z Nov 06 12:47:36 [1] dot-test-target.powerdns.org: Using DoT with 188.166.104.87:853
2024-11-06T12:48:16.2477057Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: Got 3 answers from pdns-public-ns1.powerdns.com (188.166.104.87), rcode=0 (No Error), aa=1, in 388ms
2024-11-06T12:48:16.2478614Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: Accept answer 'dot-test-target.powerdns.org|TXT|"https://github.com/PowerDNS/pdns/pull/12825"' from 'powerdns.org' nameservers? ttl=15, place=1 YES!
2024-11-06T12:48:16.2480988Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: Accept answer 'dot-test-target.powerdns.org|RRSIG|TXT 13 3 3600 20241114000000 20241024000000 13432 powerdns.org. WjjbTNTunQcpNBiXJalAIQRXnbTWfSwsW3pPxbO3kAS8mPfmzJnaN5PpUUEk1Dv5pJafk9KNpdk0rWOldQ9qKw==' from 'powerdns.org' nameservers? ttl=15, place=1 RRSIG - separate
2024-11-06T12:48:16.2483018Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: OPT answer '.' from 'powerdns.org' nameservers
2024-11-06T12:48:16.2484006Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: Got initial zone status Secure for record dot-test-target.powerdns.org|TXT
2024-11-06T12:48:16.2485193Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: Validating non-additional TXT record for dot-test-target.powerdns.org
2024-11-06T12:48:16.2485909Z Nov 06 12:47:37 [1] powerdns.org: Retrieving DNSKEYs
2024-11-06T12:48:16.2486321Z Nov 06 12:47:37 [1]  QM powerdns.org: doResolve
2024-11-06T12:48:16.2486893Z Nov 06 12:47:37 [1]  powerdns.org: Wants DNSSEC processing, auth data required by query for DNSKEY
2024-11-06T12:48:16.2487497Z Nov 06 12:47:37 [1]  QM powerdns.org: Step0 Not cached
2024-11-06T12:48:16.2488127Z Nov 06 12:47:37 [1]  QM powerdns.org: Checking if we have NS in cache for 'powerdns.org'
2024-11-06T12:48:16.2489304Z Nov 06 12:47:37 [1]  QM powerdns.org: NS (with ip, or non-glue) in cache for 'powerdns.org' -> 'pdns-public-ns2.powerdns.com', within bailiwick: 0, not in cache / did not look at cache
2024-11-06T12:48:16.2490828Z Nov 06 12:47:37 [1]  QM powerdns.org: NS (with ip, or non-glue) in cache for 'powerdns.org' -> 'pdns-public-ns1.powerdns.com', within bailiwick: 0, not in cache / did not look at cache
2024-11-06T12:48:16.2492293Z Nov 06 12:47:37 [1]  QM powerdns.org: We have NS in cache for 'powerdns.org' (flawedNSSet=0)
2024-11-06T12:48:16.2492974Z Nov 06 12:47:37 [1]  QM powerdns.org: Step1 Ancestor from cache is powerdns.org
2024-11-06T12:48:16.2493537Z Nov 06 12:47:37 [1]  QM powerdns.org: Step2 New child powerdns.org
2024-11-06T12:48:16.2494067Z Nov 06 12:47:37 [1]  QM powerdns.org: Step3 Going to do final resolve
2024-11-06T12:48:16.2494723Z Nov 06 12:47:37 [1]  powerdns.org: Wants DNSSEC processing, auth data required by query for DNSKEY
2024-11-06T12:48:16.2495621Z Nov 06 12:47:37 [1]  powerdns.org: No cache hit for 'powerdns.org|DNSKEY', trying to find an appropriate NS record
2024-11-06T12:48:16.2496462Z Nov 06 12:47:37 [1]  powerdns.org: Checking if we have NS in cache for 'powerdns.org'
2024-11-06T12:48:16.2497618Z Nov 06 12:47:37 [1]  powerdns.org: NS (with ip, or non-glue) in cache for 'powerdns.org' -> 'pdns-public-ns2.powerdns.com', within bailiwick: 0, not in cache / did not look at cache
2024-11-06T12:48:16.2499136Z Nov 06 12:47:37 [1]  powerdns.org: NS (with ip, or non-glue) in cache for 'powerdns.org' -> 'pdns-public-ns1.powerdns.com', within bailiwick: 0, not in cache / did not look at cache
2024-11-06T12:48:16.2500219Z Nov 06 12:47:37 [1]  powerdns.org: We have NS in cache for 'powerdns.org' (flawedNSSet=0)
2024-11-06T12:48:16.2500891Z Nov 06 12:47:37 [1]  powerdns.org: Cache consultations done, have 2 NS to contact
2024-11-06T12:48:16.2501792Z Nov 06 12:47:37 [1]  powerdns.org: Nameservers: pdns-public-ns2.powerdns.com(0.00ms), pdns-public-ns1.powerdns.com(0.00ms)
2024-11-06T12:48:16.2502709Z Nov 06 12:47:37 [1]  powerdns.org: Trying to resolve NS 'pdns-public-ns2.powerdns.com' (1/2)
2024-11-06T12:48:16.2503632Z Nov 06 12:47:37 [1]  pdns-public-ns2.powerdns.com: Nameserver pdns-public-ns2.powerdns.com IPs: 45.55.10.200(0.00ms)
2024-11-06T12:48:16.2504598Z Nov 06 12:47:37 [1]  powerdns.org: Resolved 'powerdns.org' NS pdns-public-ns2.powerdns.com to: 45.55.10.200
2024-11-06T12:48:16.2505434Z Nov 06 12:47:37 [1]  powerdns.org: Trying IP 45.55.10.200:53, asking 'powerdns.org|DNSKEY'
2024-11-06T12:48:16.2506026Z Nov 06 12:47:37 [1]  powerdns.org: Using DoT with 45.55.10.200:853
2024-11-06T12:48:16.2506897Z Nov 06 12:47:37 [1]  powerdns.org: Got 3 answers from pdns-public-ns2.powerdns.com (45.55.10.200), rcode=0 (No Error), aa=1, in 17ms
2024-11-06T12:48:16.2508506Z Nov 06 12:47:37 [1]  powerdns.org: Accept answer 'powerdns.org|DNSKEY|257 3 13 qOheQeq7do0x+BmuuWGuJycEXMmtTBgzsUB/bWWmIfsKD2ol8CPy6aJcLHzGR3rSlGitQJSpmcUfN/BU9m414Q==' from 'powerdns.org' nameservers? ttl=15, place=1 YES!
2024-11-06T12:48:16.2511009Z Nov 06 12:47:37 [1]  powerdns.org: Accept answer 'powerdns.org|RRSIG|DNSKEY 13 2 3600 20241114000000 20241024000000 13432 powerdns.org. xC8GwYbtS5FvH+KkAElTu4CesCoH9tADqTYl538nb8ovtHWNhg5bCdjDX9u40ulctX3IUgxHDO7lkS1hPINa1Q==' from 'powerdns.org' nameservers? ttl=15, place=1 RRSIG - separate
2024-11-06T12:48:16.2512722Z Nov 06 12:47:37 [1]  powerdns.org: OPT answer '.' from 'powerdns.org' nameservers
2024-11-06T12:48:16.2513418Z Nov 06 12:47:37 [1]  powerdns.org: Got initial zone status Secure for record powerdns.org|DNSKEY
2024-11-06T12:48:16.2514064Z Nov 06 12:47:37 [1]  powerdns.org: Validating DNSKEY for powerdns.org
2024-11-06T12:48:16.2514535Z Nov 06 12:47:37 [1]   QM powerdns.org: doResolve
2024-11-06T12:48:16.2515096Z Nov 06 12:47:37 [1]   powerdns.org: Wants DNSSEC processing, auth data required by query for DS
2024-11-06T12:48:16.2516011Z Nov 06 12:47:37 [1]   powerdns.org: Found cache hit for DS: 13432 13 2 0989896b517373685af84f704d8f24068ac5fa2a8c30147483a6dc550ad2acbf[ttl=14] 
2024-11-06T12:48:16.2516980Z Nov 06 12:47:37 [1]   powerdns.org: Updating validation state with cache content for powerdns.org to Secure
2024-11-06T12:48:16.2517629Z Nov 06 12:47:37 [1]   QM powerdns.org: Step0 Found in cache
2024-11-06T12:48:16.2518148Z Nov 06 12:47:37 [1]  powerdns.org: Trying to validate 1 DNSKEYs with 1 DS
2024-11-06T12:48:16.2518910Z Nov 06 12:47:37 [1]  powerdns.org: got valid DNSKEY (it matches the DS) with tag 13432 and algorithm 13 for powerdns.org
2024-11-06T12:48:16.2519602Z Nov 06 12:47:37 [1]  powerdns.org: We now have 1 DNSKEYs
2024-11-06T12:48:16.2520185Z Nov 06 12:47:37 [1]  powerdns.org: Validation result is Secure, current state is Indeterminate
2024-11-06T12:48:16.2521045Z Nov 06 12:47:37 [1]  powerdns.org: Validation state was Indeterminate, state update is Secure, validation state is now Secure
2024-11-06T12:48:16.2521849Z Nov 06 12:47:37 [1]  powerdns.org: Determining status after receiving this packet
2024-11-06T12:48:16.2522990Z Nov 06 12:47:37 [1]  powerdns.org: Answer is in: resolved to '257 3 13 qOheQeq7do0x+BmuuWGuJycEXMmtTBgzsUB/bWWmIfsKD2ol8CPy6aJcLHzGR3rSlGitQJSpmcUfN/BU9m414Q==|DNSKEY'
2024-11-06T12:48:16.2524017Z Nov 06 12:47:37 [1]  powerdns.org: Status=got results, this level of recursion done
2024-11-06T12:48:16.2524587Z Nov 06 12:47:37 [1]  powerdns.org: Validation status is Secure
2024-11-06T12:48:16.2525102Z Nov 06 12:47:37 [1]  QM powerdns.org: Step3 Final resolve: No Error/2
2024-11-06T12:48:16.2525646Z Nov 06 12:47:37 [1] powerdns.org: Retrieved 1 DNSKeys, state is Secure
2024-11-06T12:48:16.2526614Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: Going to validate 1 record contents with 1 sigs and 1 keys for dot-test-target.powerdns.org|TXT
2024-11-06T12:48:16.2527765Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: Signature by key with tag 13432 and algorithm ECDSAP256SHA256 was valid
2024-11-06T12:48:16.2528684Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: Validated dot-test-target.powerdns.org/TXT
2024-11-06T12:48:16.2529347Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: Secure!
2024-11-06T12:48:16.2530061Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: Validation result is Secure, current state is Secure
2024-11-06T12:48:16.2530926Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: Determining status after receiving this packet
2024-11-06T12:48:16.2532133Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: Answer is in: resolved to '"https://github.com/PowerDNS/pdns/pull/12825"|TXT'
2024-11-06T12:48:16.2533114Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: Status=got results, this level of recursion done
2024-11-06T12:48:16.2533878Z Nov 06 12:47:37 [1] dot-test-target.powerdns.org: Validation status is Secure
2024-11-06T12:48:16.2534579Z Nov 06 12:47:37 [1] QM dot-test-target.powerdns.org: Step3 Final resolve: No Error/2
2024-11-06T12:48:16.2535920Z Nov 06 12:47:37 msg="Validates Correctly" subsystem="syncres" level="0" prio="Info" tid="2" ts="1730897257.163" ecs="" mtid="1" proto="udp" qname="dot-test-target.powerdns.org" qtype="TXT" remote="127.0.0.1:47204" vstate="Secure"
2024-11-06T12:48:16.2538705Z Nov 06 12:47:37 msg="Answer" subsystem="syncres" level="0" prio="Info" tid="2" ts="1730897257.163" additional="1" answer-is-variable="0" answers="2" dotout="3" ecs="" into-packetcache="1" maxdepth="2" mtid="1" netms="2309.528000" outqueries="10" proto="udp" qname="dot-test-target.powerdns.org" qtype="TXT" rcode="0" rd="1" remote="127.0.0.1:47204" tcpout="0" throttled="0" timeouts="1" totms="2324.583000" validationState="Secure"
2024-11-06T12:48:16.2540512Z Nov 06 12:47:38 Exiting on user request
2024-11-06T12:48:16.2541163Z Nov 06 12:47:38 msg="Unloaded current lua script" subsystem="runtime" level="0" prio="Info" tid="1" ts="1730897258.172" name=""
2024-11-06T12:48:16.2542106Z Nov 06 12:47:38 msg="Unloaded current lua script" subsystem="runtime" level="0" prio="Info" tid="2" ts="1730897258.172" name=""
2024-11-06T12:48:16.2543056Z Nov 06 12:47:38 msg="Unloaded current lua script" subsystem="runtime" level="0" prio="Info" tid="3" ts="1730897258.173" name=""
2024-11-06T12:48:16.2544006Z Nov 06 12:47:38 msg="Unloaded current lua script" subsystem="runtime" level="0" prio="Info" tid="4" ts="1730897258.173" name=""
2024-11-06T12:48:16.2544546Z 
2024-11-06T12:48:16.2544711Z ==============> configs/SimpleDoT/recursor.log <==============
```

### Other information
<!-- if you already did more digging into the issue, please provide all the information you gathered -->

--- a/regression-tests.recursor-dnssec/test_SimpleDoT.py
+++ b/regression-tests.recursor-dnssec/test_SimpleDoT.py
@@ -35,7 +35,8 @@ devonly-regression-test-mode
         query = dns.message.make_query('dot-test-target.powerdns.org', 'TXT', want_dnssec=True)
         query.flags |= dns.flags.AD
 
-        res = self.sendUDPQuery(query)
+        # As this test uses external servers, be a more generous wrt timeouts than the default 2.0s
+        res = self.sendUDPQuery(query, timeout=5.0)
 
         self.assertMessageIsAuthenticated(res)
         self.assertRRsetInAnswer(res, expected)

